examples/platforms/kw41z/CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+set(OT_PLATFORM_LIB "openthread-kw41z" PARENT_SCOPE)
+
+if(NOT OT_CONFIG)
+    set(OT_CONFIG "openthread-core-kw41z-config.h")
+    set(OT_CONFIG ${OT_CONFIG} PARENT_SCOPE)
+endif()
+
+list(APPEND OT_PLATFORM_DEFINES
+    "OPENTHREAD_CORE_CONFIG_PLATFORM_CHECK_FILE=\"openthread-core-kw41z-config-check.h\""
+    "OPENTHREAD_SETTINGS_RAM=0"
+)
+set(OT_PLATFORM_DEFINES ${OT_PLATFORM_DEFINES} PARENT_SCOPE)
+
+list(APPEND OT_PLATFORM_DEFINES "OPENTHREAD_PROJECT_CORE_CONFIG_FILE=\"${OT_CONFIG}\"")
+
+add_library(openthread-kw41z
+    alarm.c
+    diag.c
+    entropy.c
+    flash.c
+    misc.c
+    radio.c
+    system.c
+    logging.c
+    uart.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/startup_MKW41Z4.S
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/clock_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/system_MKW41Z4.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_clock.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_flash.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_lpuart.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_gpio.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_pit.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/drivers/fsl_trng.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/dbg_ram_capture.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/fsl_xcvr.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/fsl_xcvr_trim.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/ifr_radio.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_ant_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_ble_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_common_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p3_h_0p5_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p5_h_0p32_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p5_h_0p5_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p5_h_0p7_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p5_h_1p0_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_gfsk_bt_0p7_h_0p5_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_mode_datarate_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_msk_config.c
+    ${PROJECT_SOURCE_DIR}/third_party/nxp/MKW41Z4/XCVR/cfgs_kw4x_3x_2x/fsl_xcvr_zgbe_config.c
+    $<TARGET_OBJECTS:openthread-platform-utils>
```

Comment:
Same with [comment](https://github.com/openthread/openthread/pull/4691#discussion_r396167401) in #4691.